notebooks/community/model_evaluation/automl_tabular_regression_model evaluation.ipynb

```diff
@@ -0,0 +1,1273 @@
+{
```

Comment:
<p>Add text</p><p> </p><p>Run the following commands to grant your service account access to read and write pipeline artifacts in the bucket that you created in the previous step. You only need to run this step once per service account.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>